UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -305,7 +305,6 @@ public virtual void AddBodyPartSlot(ItemSlot ItemSlot)
 		/// Removes the Body Part Item from the storage of its parent (a body part container or another body part)
 		/// Will check if the this body part causes death upon removal and will tint it's Item Sprite to the character's skinTone if allowed.
 		/// </summary>
-		[ContextMenu("Debug - Drop this body part.")]
```

Comment:
I was going to say "Because `DismemberBodyPartLogic()` seems to be the better function to hold this so we don't have issues where people remove torsos then complain about walking around torsoless even though they caused this issue themselves." but then I took a good look at it and I think we can keep that gib check inside `RemoveBodyPartThis()` and the other function that replaced it is no longer needed.

Reply:
This decorator is to make the function available in unity editor, to debug, nothing more. No one should complain about losing limbs when opening editor debug tab and clicking the function.

Reply:
The key word here is _no one should_ but people who are debugging this might accidently think this is intended behavior and complain about it. So I'm making the logic consistent even for the people who are debugging.